web-common/src/features/dashboards/leaderboard/LeaderboardMeasureSelector.svelte

```diff
@@ -81,6 +106,53 @@
   const setAllDimensionsVisible = () => {
     showHideDimensions.setAllToVisible();
   };
+
+  let disabledButtons: ("delta" | "pie")[] = [];
```

Comment:
Should we move this to a separate component? Showing percents doesnt fit within LeaderboardMeasureSelector.